exercises/hello-world/src/HelloWorld.purs

```diff
@@ -0,0 +1,6 @@
+module HelloWorld where
+
+import Prelude
+import Data.Maybe (Maybe(Just, Nothing))
+
+helloWorld :: Maybe String -> String
```

Comment:
I think the question for me is whether the student should be working out the type sig themselves or whether we should provide it.